php/libraries/BVL_Feedback_Panel.class.inc

```diff
@@ -66,7 +66,63 @@ class BVL_Feedback_Panel
                 $commentID
             );
     }
+    /**
+     * Returns all the sourcefrom instruments from parameter_type
+     *
+     * @param string|null $instrument If specified, return fields from this
+     *                                test_name
+     * @param string|null $commentID  If specified, return fields for this
+     *                                commentid
+     * @param string|null $name       If specified, return fields for this
+     *                                parameter_type name
+     *
+     * @return  ?array Format: array(
+     *              0 => array(
+     *                   'SourceField' => value
+     *                   'Name'        => name
+     *
+     *              )
+     *         )
+     * @note    Function comment written by Dave, not the author of this function.
+     * @cleanup
+     */
+    static function getSourcefields(
```

Comment:
This move also seems arbitrary. The `parameter_type` table is the data dictionary, it's not just for the feedback panel.